AttributeError: '_FileEq' object has no attribute '_update_message'

To reproduce:

```
sirepo (master)$ echo '{}' > ~/src/radiasoft/sirepo/tests/lib_data/elegant_1/out.json
sirepo (master)$ pykern test tests/lib_test.py  case=elegant
tests/lib_test.py FAIL tests/lib_test.log
============================================================= test session starts ==============================================================
platform linux -- Python 3.9.15, pytest-7.2.1, pluggy-1.0.0 -- /home/vagrant/.pyenv/versions/3.9.15/envs/py3/bin/python
cachedir: .pytest_cache
rootdir: /home/vagrant/src/radiasoft/sirepo
collecting ... collected 2 items / 1 deselected / 1 selected

tests/lib_test.py::test_elegant FAILED

=================================================================== FAILURES ===================================================================
_________________________________________________________________ test_elegant _________________________________________________________________
Traceback (most recent call last):
  File "/home/vagrant/src/radiasoft/sirepo/tests/lib_test.py", line 13, in test_elegant
    _code()
  File "/home/vagrant/src/radiasoft/sirepo/tests/lib_test.py", line 42, in _code
    pkunit.file_eq(s.join("out.json"), d2)
  File "/home/vagrant/src/radiasoft/pykern/pykern/pkunit.py", line 268, in file_eq
    _FileEq(expect_path, *args, **kwargs)
  File "/home/vagrant/src/radiasoft/pykern/pykern/pkunit.py", line 509, in __init__
    self._compare()
  File "/home/vagrant/src/radiasoft/pykern/pykern/pkunit.py", line 608, in _compare
    pkfail("expect != actual:\n{}", _failed_msg(p))
  File "/home/vagrant/src/radiasoft/pykern/pykern/pkunit.py", line 573, in _failed_msg
    return r + self._update_message
AttributeError: '_FileEq' object has no attribute '_update_message'
======================================================= 1 failed, 1 deselected in 1.70s ========================================================
```

--- a/pykern/pkunit.py
+++ b/pykern/pkunit.py
@@ -651,6 +651,10 @@ the expect jinja template={self._expect_path} manually.
             if self.is_bytes
             else (pkio.read_text, pkio.write_text)
         )
+        self._update_message = f"""
+to update test data:
+        cp '{self._actual_path}' '{self._expect_path}'
+"""
         if self._expect_csv():
             assert not self.is_bytes, "csv is not compatible with is_bytes"
             return
@@ -668,10 +672,6 @@ the expect jinja template={self._expect_path} manually.
             assert not self.is_bytes, "json or jinja is not compatible with is_bytes"
             return
         self._expect_default()
-        self._update_message = f"""
-to update test data:
-        cp '{self._actual_path}' '{self._expect_path}'
-"""
 
     def _validate_args(self, expect_path, *args, **kwargs):
         from pykern.pkcollections import PKDict

